app/authentication/authenticator.py

```diff
@@ -28,14 +33,25 @@ def request_load_user(request):  # pylint: disable=unused-argument
     return load_user()
 
 
+@user_logged_out.connect_via(ANY)
+def when_user_logged_out(sender_app, user):  # pylint: disable=unused-argument
```

Comment:
Could you not silence these by just adding a _prefix before each variable passed instead wherever we pass unused arguments?

Reply:
I tried this and got an error when this method is called with a keyword argument that doesn't match the `_` prefixed parameters:
`when_user_logged_out() got an unexpected keyword argument 'user'`